content/about/limits.md

```diff
@@ -1,44 +1,78 @@
-﻿---
+---
 title: Limits
 weight: 4
 ---
 
+- [Plans](#plans)
+- [Script Size](#script-size)
+- [Number of Scripts](#number-of-scripts)
+- [Request Limits](#request-limits)
+  - [Burst Rate Limit](#burst-rate-limit)
+  - [Daily Request Limit](#daily-request-limit)
+     - [Fail Open](#fail-open)
+     - [Fail Closed](#fail-closed)
+- [CPU/Execution Time Limit](#cpu-execution-time-limit)
+- [Memory](#memory)
+- [Subrequests](#subrequests)
+- [Simultaneous Open Connections](#simultaneous-open-connections)
+  
 ## Plans
 
-Workers plans are separate from any Cloudflare plan (Free, Professional, Business) you may have. Everyone has access to the Free tier Workers platform by default. Upgrading to the Unlimited (Paid) plan will affect the number of requests your scripts can handle, as well as the compute time available per request.
+Workers plans are separate from any Cloudflare plan (Free, Professional, Business) you may have. Everyone has access to the Free tier Workers platform by default and are subject to the [free tier request limits](#request-limits). Upgrading to the Unlimited (Paid) plan lifts the restrictions and increases your CPU time limit.
+
+| Plan             | [CPU Limit](#cpu-execution-time-limit) | [Daily Request Limit](#daily-request-limit) | [Burst Rate Limit](#burst-rate-limit) |
+| ---------------- | --------------------------------------- | ------------------------------------------- | ------------------------------------- |
+| Free             | 10ms                                    | 100,000                                     | 1000 requests per 10 min              |
+| Unlimited (Paid) | 50ms                                    | none                                        | none                                  |
+
+See definitions for the types of limits and behavior below.
+Script size, number of scripts, subrequests, and available memory are not affected by plan type.
 
 ## Script Size
 
-A Workers script plus any [Asset Bindings](/tooling/api/bindings) can be up to 1MB in size after compression.
+A Workers script plus any [Asset Bindings](/tooling/api/bindings) can be up to **1MB** in size after compression.
 
 ## Number of Scripts
 
-Unless otherwise negotiated as a part of an enterprise level contract, all Workers accounts are limited to a maximum of 30 scripts at any given time.
+Unless otherwise negotiated as a part of an enterprise level contract, all Workers accounts are limited to a maximum of **30 scripts at any given time**.
 
-**Note:** app Workers scripts do not count towards this limit.
+**Note:** App Workers scripts do not count towards this limit.
```

Comment:
maybe out of scope for this pr but maybe we should add a link (if there is one) to app workers?

Reply:
Meh we are not encouraging them to find out about App Workers. I actually thought about removing this all together but it is useful reference documentation